app/controllers/reactions_controller.rb

```diff
@@ -101,5 +101,12 @@ def remove_count_cache_key
     return unless params[:reactable_type] == "Article"
 
     Rails.cache.delete "count_for_reactable-Article-#{params[:reactable_id]}"
+    article = Article.find_by(id: params[:reactable_id])
```

Comment:
Basically there are two states on the `multiple_reactions_engagement` element that are stored in the initial request: Taking up space, or not.

We need to bust the cache on the initial creation of the first reaction to flip this state, but we don't want to bust the cache on all subsequent reactions (they are loaded async).

There is a small inefficiency here where it busts on either 0 or 1, but that accounts for states of going up, or down, because this is called before the non-idempotent action.